readthedocs/rtd_tests/tests/test_project.py

```diff
@@ -32,13 +32,6 @@ def setUp(self):
 
 class TestProject(ProjectMixin, TestCase):
 
-    def test_valid_versions(self):
```

Comment:
This test is always failing because this is calling `valid_versions` which checks for `num_major` and `num_minor`, which are deleted. Hence this was always failing.